WordPress/src/androidTest/java/org/wordpress/android/util/encryption/EncryptionUtilsTest.java

```diff
@@ -77,6 +80,29 @@ public void testCharacterSampleEncryptionResultIsValid() {
         testEncryption(TEST_CHAR_SAMPLE);
     }
 
+    @Test
+    public void testUUIDAreDifferent() {
+        String source1 = "v1" + "android" + System.currentTimeMillis();
+        String uuid1 = UUID.nameUUIDFromBytes(source1.getBytes()).toString();
+
+        String source2 = "v1" + "android" + System.currentTimeMillis();
+        String uuid2 = UUID.nameUUIDFromBytes(source2.getBytes()).toString();
+
+        assertFalse(uuid1.equals(uuid2));
```

Comment:
It seems to me these two tests are testing the implementation of `UUID.nameUUIDFromBytes` more than they are testing the functionality of this class. WDYT?

Reply:
Yes, you are totally right. :) Good spot! 

Actually, I created them while playing with UUID to be able to test the result from `UUID.nameUUIDFromBytes` running multiple times.  I should remove them and update the existing tests to accommodate the new logic.